LICENSE

```diff
@@ -1,6 +1,7 @@
 MIT License
```

Comment:
We use the Apache 2 license for our other open projects. I think we should try to stay with one.
Suggest we copy the original license to a file called LICENSE-fake-sftp-server-lambda and convert this to a Apache 2 license file.